docs/data-guidelines.md

```diff
@@ -12,6 +12,17 @@ If it's helpful to understanding the rule, summarize the rationale. Definitely c
 
 -- END TEMPLATE -->
 
+
+## Adding subfeatures
```

Comment:
@queengooborg @foolip OK, I have reworded this to be the opposite. This says "always add a subfeature for any entity or behaviour where there is an implementation in a browser."